packages/compass-settings/src/components/settings/feature-preview.tsx

```diff
@@ -92,30 +80,4 @@ export const FeaturePreviewSettings: React.FunctionComponent<{
   );
 };
 
-export default withPreferences(
-  connect(
-    (
-      state: RootState,
-      ownProps: {
-        enableAIExperience?: boolean;
-        enableAIWithoutRolloutAccess?: boolean;
-        cloudFeatureRolloutAccess?: UserPreferences['cloudFeatureRolloutAccess'];
-      }
-    ) => {
-      return {
-        showAtlasLoginSettings:
-          state.settings.settings.enableAIExperience ||
-          ['authenticated', 'in-progress'].includes(state.atlasLogin.status) ||
```

Comment:
Now that we have a bunch of people logged in but with feature disabled I guess we can't really rely on login state be good enough indicator to show atlas login feature, even if it means that they wouldn't be able to log out manually